src/plugins/intel_cpu/src/nodes/gather_compression.cpp

```diff
@@ -0,0 +1,224 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gather_compression.h"
+
+#include <openvino/op/gather.hpp>
+#include <openvino/op/constant.hpp>
+
+#include <string>
+#include <vector>
+
+#include "openvino/core/parallel.hpp"
+#include <openvino/opsets/opset1.hpp>
+#include "common/cpu_memcpy.h"
+#include "common/cpu_convert.h"
+#include "utils/general_utils.h"
+#include "kernels/x64/gather_uni_kernel.hpp"
+#include <partitioned_mem_mgr.h>
+#include "shape_inference/custom/gather.hpp"
+#include "utils/ngraph_utils.hpp"
+#include "transformations/cpu_opset/common/op/gather_compression.hpp"
+#include "memory_desc/dnnl_blocked_memory_desc.h"
```

Comment:
there are some header files seem to be un-necessary (`kernels/x64/gather_uni_kernel.hpp` for example?) , please check and remove them